velox/functions/prestosql/types/IPPrefixType.cpp

```diff
@@ -70,6 +80,12 @@ class IPPrefixCastOperator : public exec::CastOperator {
     switch (resultType->kind()) {
       case TypeKind::VARCHAR:
         return castToString(input, context, rows, *result);
+      case TypeKind::HUGEINT: {
+        if (isIPAddressType(input.type())) {
+          return castToIpAddress(input, context, rows, *result);
```

Comment:
Can return an Fail. Something like this
```
VELOX_FAIL("We should be using IPPREFIX castTo function");
```

Or can just do nothing. It should never be called anyways since the castFrom calls come first.

Reply:
Ah good catch, will update